functionList/LSL_byKimpaTammas.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- ==========================================================================\
+|
+|   To learn how to make your own language parser, please check the following
+|   link:
+|       https://npp-user-manual.org/docs/function-list/
+|
+\=========================================================================== -->
+<NotepadPlus>
+	<functionList>
+		<!-- ================================================================= [ LSL ] -->
+		<!-- LSL - Linden Script Language                                              -->
+
+		<parser
+			id="lsl"
+			displayName="LSL"
+			commentExpr="(?:(/\*.*?\*/)|(//.*?$))"
+		>
+			<classRange
+				mainExpr="(?x)(?:\b(?:(state)\s+(?:[A-Za-z_]\w*)|(default))\b)(?=\s*\{)"

```

Comment:
Superfluous parentheses around `state` and around `default`.
Numbered and named capturing groups are not supported.